docs/my-website/docs/mcp.md

```diff
@@ -180,9 +180,12 @@ You can choose to access specific MCP servers and only list their tools using th
 - Limit tool access to one or more specific MCP servers
 - Control which tools are available in different environments or use cases
 
-The header accepts a JSON array of server names, where:
+The header accepts either:
+1. A comma-separated list of server names: `"Zapier_Gmail,Server2,Server3"`
+2. A JSON array of server names (legacy format): `["Zapier_Gmail", "Server2", "Server3"]`
```

Comment:
just allow one way to use this which is csv. Don't maintain the JSON array

Reply:
ack